src/shared/components/post/post-listing.tsx

```diff
@@ -33,9 +33,20 @@ import { PostForm } from "./post-form";
 import { PostListingList } from "./post-listing-list";
 import { PostListingCard } from "./post-listing-card";
 import { masonryUpdate } from "@utils/browser";
+import {
+  areKeyboardShortcutsEnabled,
+  shouldIgnoreEvent,
+  activateKeyboardMode,
+} from "@utils/keyboard-shortcuts";
+import {
+  handleKeyboardShortcut,
+  type KeyboardShortcutHandlers,
+} from "@utils/keyboard-shortcuts-actions";
+import classNames from "classnames";
 
 type PostListingState = {
   showEdit: boolean;
+  imageExpanded: boolean;
```

Comment:
We removed all imageExpanded, and now just use postBodyType (list, card, smallcard), so you can get rid of this.